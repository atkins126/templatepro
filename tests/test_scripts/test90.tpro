{{loop(objectsb)}}
Prop1 = {{objectsb.Prop1}}
{{if(objectsb.Prop1)}}
Prop1 is TRUE
{{else}}
Prop1 is FALSE
{{endif}}
Prop2 = {{objectsb.Prop2}}
{{if(objectsb.Prop2)}}
Prop2 is TRUE
{{else}}
Prop2 is FALSE
{{endif}}
Prop3 = {{objectsb.Prop3}}
{{if(objectsb.Prop3)}}
Prop3 is TRUE
{{else}}
Prop3 is FALSE
{{endif}}
PropInt = {{objectsb.PropInt}}
{{if(objectsb.PropInt)}}
PropInt is TRUE
{{else}}
PropInt is FALSE
{{endif}}

----------------------------

{{endloop}}