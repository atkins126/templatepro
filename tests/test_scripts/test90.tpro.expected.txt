Prop1 = true
Prop1 is TRUE
Prop2 = false
Prop2 is FALSE
Prop3 = value3.1
Prop3 is TRUE
PropInt = 0
PropInt is FALSE

----------------------------

Prop1 = false
Prop1 is FALSE
Prop2 = true
Prop2 is TRUE
Prop3 = value3.2
Prop3 is TRUE
PropInt = 1
PropInt is TRUE

----------------------------

Prop1 = 1
Prop1 is TRUE
Prop2 = 0
Prop2 is FALSE
Prop3 = value3.3
Prop3 is TRUE
PropInt = 0
PropInt is FALSE

----------------------------

